### PR TITLE
Aos types 

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: aostypes/aostypes.go [hunk 1]
```diff
@@ -113,6 +113,50 @@ type BoardConfig struct {
...
 }
 
+// ServiceInfo service info.
+type ServiceInfo struct {
+	VersionInfo
+	ID         string `json:"id"`
+	ProviderID string `json:"providerId"`
+	URL        string `json:"url"`
+	Sha256     []byte `json:"sha256"`
+	Sha512     []byte `json:"sha512"`
```
Comment (al1img) [inline aostypes/aostypes.go:123]: Do we need size as well?

File: aostypes/aostypes.go [hunk 2]
```diff
@@ -113,6 +113,50 @@ type BoardConfig struct {
...
+}
+
+// LayerInfo layer info.
+type LayerInfo struct {
+	VersionInfo
+	ID     string `json:"id"`
+	Digest string `json:"digest"`
+	URL    string `json:"url"`
+	Sha256 []byte `json:"sha256"`
+	Sha512 []byte `json:"sha512"`
```
Comment (al1img) [inline aostypes/aostypes.go:133]: Size?

File: aostypes/aostypes.go (hunk 1 above)
Comment (Eabramov84) [reply aostypes/aostypes.go:123]: yes. missed by misstake

File: aostypes/aostypes.go (hunk 2 above)
Comment (Eabramov84) [reply aostypes/aostypes.go:133]: yes. missed by misstake

**Review by MykolaSolyanko — APPROVED** [review]

**Review by al1img — APPROVED** [review]

**Pull request merged**